corehq/apps/toggle_ui/views.py

```diff
@@ -149,6 +167,61 @@ def post(self, request, *args, **kwargs):
             clear_toggle_cache(toggle.slug, item)
 
         data = {
-            'item_list': item_list
+            'items': item_list
         }
+        if self.usage_info:
+            data['last_used'] = _get_usage_info(toggle)
         return HttpResponse(json.dumps(data), content_type="application/json")
+
+
+def _get_usage_info(toggle):
+    """Returns usage information for each toggle
+    """
+    last_used = {}
+    for enabled in toggle.enabled_users:
+        name = _enabled_item_name(enabled)
+        try:
+            if _namespace_domain(enabled):
+                last_used[name] = _format_date(get_last_form_submission_received(name))
```

Comment:
non-blocking: looks like `get_last_form_submission_received()` does not raise `ResourceNotFound`, so the `try`/`except` could be moved into the `else` block. Put another way, this `if` branch could be moved out of the the `try`/`except`.